xyz-jobs/xyz-job-service/src/main/java/com/here/xyz/jobs/datasets/filters/SpatialFilter.java

```diff
@@ -62,6 +64,11 @@ public SpatialFilter withRadius(final int radius) {
     return this;
   }
 
+  public boolean isValidGeom()
```

Comment:
That seems to be redundant. It would be even serialized to the user if being kept.